lib/content-services/src/lib/node-comments/services/node-comments.service.spec.ts

```diff
@@ -81,5 +84,44 @@ describe('NodeCommentsService', () => {
                 responseText: JSON.stringify(fakeContentComments)
             });
         });
+
+        it('should cache the avatar URL when new comment is processed', (done) => {
+            spyOn(contentService, 'getContentUrl').and.returnValue('fake-url');
+
+            service.add('999', fakeContentComment.entry.content).subscribe((result: CommentModel) => {
+                expect(contentService.getContentUrl).toHaveBeenCalledWith('123-123-123');
+                expect(service.getUserImage('fake-email@dom.com')).toBe('fake-url');
+                expect(result.createdBy.id).toBe('fake-email@dom.com');
+                done();
+            });
+
+            jasmine.Ajax.requests.mostRecent().respondWith({
+                status: 200,
+                contentType: 'application/json',
+                responseText: JSON.stringify(fakeContentComment)
+            });
+        });
+
+        it('should return avatar URL from cache via getUserImage()', (done) => {
```

Comment:
I have feeling that this test tests the same scenario like previous test. Is that right? Looking at titles both are testing scenario when avatar is cached. So if both test the same scenario can we combine both tests into single test to avoid repeating?